src/Internal/Service/PaymentFraudPreventionService.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Class PaymentFraudPreventionService
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Service;
+
+use WC_Payments_Account;
+
+/**
+ * Class PaymentFraudPreventionService
+ */
+class PaymentFraudPreventionService {
```

Comment:
Any reason not to just call it `FraudPreventionService`?
```suggestion
class FraudPreventionService {
```

Reply:
Just my own preference to let it a bit more specific when seeing a few other services in `includes/fraud-prevent` folder. 

I am not too adherent here so will make change except that makes sense to you too.